andela/ah-frontend-athena
# #162163220 SetUp folder/file structure for redux

**What does this PR do?**
This Pull Request sets up the folder/file structure for `Redux`
**Description of Task to be completed?**
We anticipate `Authors Heaven` to be a huge project so we need to integrate `Redux` into our front-end `React` app for proper state management and easy maintainability
**How should this be manually tested?**
- Install the `Redux Devtools` chrome extension
- Clone the repo
- Check out the `ch-setup-redux-162163220` branch
- run `npm` install
- run `npm start` to start the development server
- confirm that the `Redux Devtools` extension is activated meaning the application is using redux

**What are the relevant pivotal tracker stories?**
#162163220
